src/qt/transactiondesc.cpp

```diff
@@ -36,13 +36,17 @@ QString TransactionDesc::FormatTxStatus(const interfaces::WalletTxStatus& status
 {
     int depth = status.depth_in_main_chain;
     if (depth < 0) {
+        //: This status represents an unconfirmed transaction that conflicts with another
         return tr("conflicted with a transaction with %1 confirmations").arg(-depth);
     } else if (depth == 0) {
         const QString abandoned{status.is_abandoned ? QLatin1String(", ") + tr("abandoned") : QString()};
+        //: This status represents an unconfirmed transaction that is in the memory pool or possibly abandoned
         return tr("0/unconfirmed, %1").arg(inMempool ? tr("in memory pool") : tr("not in memory pool")) + abandoned;
```

Comment:
A translator comments applied to a line with multiple independent translatable strings will only apply to the first one. You can check what translator comment is applying to what string by making the translations, see more here: [translation_process.md](https://github.com/bitcoin/bitcoin/blob/master/doc/translation_process.md)

You could break this into multiple lines in order to give translator comments to the strings `in memory pool` and `not in memory pool`. Seems useful to provide context for this strings.

Reply:
I think I addressed your suggestion in 62ab92b.

Reply:
Moved this commit to PR https://github.com/bitcoin-core/gui/pull/583.